test/manual/testplan.md

```diff
@@ -0,0 +1,90 @@
+# OpenCloud Desktop Client Manual Test Plan
+
+## Account Management
+
+Try everything in `Setup and Configuration` once with Keycloak and once with Lico (the single binary builtin IdP).
+
+### Setup and Configuration
+* Add a new account with valid credentials
+* Attempt to add an account with invalid credentials
+* Skip sync folder configuration during setup
+* Configure multiple sync connections for one account
+
+### Connection Management
+* Pause and resume synchronization
```

Comment:
Maybe:
* Force a sync while an Upload to a different space is running
* Force a sync while a Download to a different space is running

Reply:
added 👍